bsp/nrf5x/nrf52840/applications/application.c

```diff
@@ -25,24 +25,154 @@
 #include <shell.h>
 #endif
 
-#include <nrf_gpio.h>
+#include <drv_gpio.h>
```

Comment:
please use the original application
just change the
api nrf_gpio_pin_set  to rt_pin_write。
you can see the file on bsp\stm32\stm32f072-st-nucleo\applications\main.c
just the same。
the gpio test code，is no needed，just for test by yourself。